core/drivers/imx_wdog.c

```diff
@@ -116,6 +112,10 @@ static TEE_Result imx_wdog_base(vaddr_t *wdog_vbase)
 		"/soc/aips-bus@2000000/wdog@20c0000",
 	};
 #endif
+	ssize_t sz = 0;
+	int off = 0;
+	int st = 0;
+	uint32_t i = 0;
```

Comment:
Why have you moved variables here?

Reply:
I think this is an artifact of how I split up the commit, since the ifdef lines above are removed in the next commit.